public-ui/src/app/views/pm/pm.component.html

```diff
@@ -55,6 +55,5 @@ <h5 class="box-title"> {{selectedGroup.groupName}}</h5>
 
   </div>
   <div class="col-xs-12 col-sm-12 col-md-3">
-    <img [src]="pageImage" />
```

Comment:
pageImage is a property somewhere, right? Unused now? Also can this image resource be deleted now?

Reply:
Removed page-image property and unused resources.